ra/next/ra-api.ng

```diff
@@ -67,10 +67,12 @@
 		<element name="action">
 			<attribute name="name">
 				<choice>
+					<!-- standard actions -->
 					<value>start</value>
 					<value>stop</value>
 					<value>recover</value>
 					<value>status</value>
+					<value>monitor</value>
```

Comment:
It's nice to have a list of standard actions, but the 1.0 standard says: "additional actions may be supported by the script for example for LSB compliance". This implies that *any* action should be allowed. That also makes sense in the context of implementations such as pacemaker defining new actions (promote, demote, notify, migrate_to, migrate_from). I think for the 1.1. RNG we should just make this a text field (or define a type if we want to limit the characters that can be used).

Reply:
1.0 standard defines also a list of available options :)

I'm fine with changing it to anything, it will also simplify changes for fence-agents API